WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Summary Section/Edit Order Status/OrderStatusListViewController.swift

```diff
@@ -104,7 +105,12 @@ extension OrderStatusListViewController {
         navigationItem.setRightBarButton(rightBarButton, animated: false)
         deActivateApplyButton()
     }
-
+    func activateRightSpinner() {
+        let activityIndicator = UIActivityIndicatorView(frame: CGRect(x: 0, y: 0, width: 20, height: 20))
+        let rightBarButton = UIBarButtonItem(customView: activityIndicator)
+        navigationItem.setRightBarButton(rightBarButton, animated: true)
+        activityIndicator.startAnimating()
+    }
```

Comment:
```suggestion

   func activateRightSpinner() {
        let activityIndicator = UIActivityIndicatorView(style: .medium)
        let rightBarButton = UIBarButtonItem(customView: activityIndicator)
        navigationItem.setRightBarButton(rightBarButton, animated: true)
        activityIndicator.startAnimating()
    }

```

Changes:
- We don't need to give the spinner a specific frame
- Please add new lines between methods